superset/cachekeys/api.py

```diff
@@ -40,10 +42,98 @@ class CacheRestApi(BaseSupersetModelRestApi):
     class_permission_name = "CacheRestApi"
     include_route_methods = {
         "invalidate",
+        "warm_up_cache",
     }
 
     openapi_spec_component_schemas = (CacheInvalidationRequestSchema,)
 
+    @expose("/warm_up_cache", methods=["GET"])
```

Comment:
this should be a `PUT` or `PATCH` since the state changes. Use the query parameters as the payload

Reply:
moved to `PUT` and query parameters to json body payload validated by marshmallow